app/src/main/res/values-in/strings.xml

```diff
@@ -73,6 +73,7 @@
   <string name="label_menu_block_ads">Blokir Iklan</string>
   <string name="label_menu_block_image">Blokir gambar</string>
   <string name="label_menu_settings">Setelan</string>
+  <string name="label_menu_exit">Keluar</string>
```

Comment:
We have these localized through https://pontoon.mozilla.org/id/firefox-rocket/app.po/ Let's remove this and we may join the localization discussion in pontoon when the string lands there.

Reply:
So for the time being it will used the English value, right?

Reply:
Yeah simply leave the `strings.xml` in `values-in/` as-is should be good enough. (Only update the one in `values/` ) :)